src/config.rs

```diff
@@ -55,53 +53,33 @@ impl Configuration {
         allowed_fns: HashMap<AppId, AllowedFns>,
         max_app_connections: &str,
         zome_call_timeout: &str,
-    ) -> HcHttpGatewayResult<Self> {
-        let admin_ws_url = Url2::try_parse(admin_ws_url).map_err(|e| {
-            HcHttpGatewayError::ConfigurationError(format!("Url parse error: {}", e))
-        })?;
+    ) -> anyhow::Result<Self> {
```

Comment:
I think if the configuration cannot be created, the error can be returned as is as a string.

Reply:
Solid case for an "Other". Nothing you can do in response so an other with a context string does seem appropriate

Reply:
I suppose you meant `std::io::Error::other`. The configuration `try_new` is only used when taking all the input from the console and parsing it https://github.com/holochain/hc-http-gw/pull/25/commits/e9c81a55b96d531775aab730e954f988136b34dc

Reply:
Nice, I like that logic